frontend/assets/style/components/_cards.scss

```diff
@@ -19,7 +19,7 @@
 }
 
 .is-dark-theme .card {
-  box-shadow: none;
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.25);
```

Comment:
Why did you make this change?

As far as I can tell this is not approved by the designs.

How this looks:

![Screen Shot 2023-04-28 at 3 31 38 PM](https://user-images.githubusercontent.com/138706/235264524-27cc9e7d-4673-4d18-a6c3-96bd347d1ab0.jpg)

How it should look:

![Screen Shot 2023-04-28 at 3 31 44 PM](https://user-images.githubusercontent.com/138706/235264527-66ce5f5d-54b7-4dbb-8d9b-bf1ac01cbc6d.jpg)